modules/cli/src/main/scala/coursier/CoursierUtil.scala

```diff
@@ -0,0 +1,9 @@
+package coursier
+
+import coursier.util.WebPage
+
+case object CoursierUtil {
```

Comment:
NIT (no need to make this a `case object`)
```suggestion
object CoursierUtil {
```